operations/mimir-mixin/alerts/alerts.libsonnet

```diff
@@ -226,8 +226,8 @@
           alert: $.alertName('RingMembersMismatch'),
           expr: |||
             (
-              avg by(%(alert_aggregation_labels)s) (sum by(%(alert_aggregation_labels)s, %(per_instance_label)s) (cortex_ring_members{name=~"%(component)s",job=~"%(job)s"}))
-              != sum by(%(alert_aggregation_labels)s) (up{job=~"%(job)s"})
+              avg by(%(alert_aggregation_labels)s) (sum by(%(alert_aggregation_labels)s, %(per_instance_label)s) (cortex_ring_members{name=~"%(job)s",job=~"(.*/)?%(job)s"}))
```

Comment:
> name=~"%(job)s"

This doesn't look correct to me. Shouldn't be the value of `component_job[0]`?

Reply:
that's what it was previously, but I think using just the component name should be fine based off a quick couple of test queries. I was just less familiar with the history of that label to be confident. Will update :-).